server/block/break_info.go

```diff
@@ -15,6 +16,12 @@ type Breakable interface {
 	BreakInfo() BreakInfo
 }
 
+// PostBreakable represents a block that has an action(s) processed after the block is broken.
+type PostBreakable interface {
+	// PostBreak is called after the block has broken.
+	PostBreak(pos cube.Pos, w *world.World, u item.User)
```

Comment:
Would do some renaming here. Maybe just `Break()` as the method name and `Breaker` or something as the interface name?

Reply:
It's @JustTalDevelops's naming scheme, but sure

Reply:
`Breaker` seems like a terrible name in the context of a broken block

Reply:
`PostBreakable` I'm not a fan of either. At least `Breaker` follows the Go standards of <method name> + `er`.

Reply:
If you have any other ideas, do feel free to list them.

Reply:
Resolved on discord